report/backcompat.go

```diff
@@ -0,0 +1,153 @@
+package report
+
+// Backwards-compatibility: code to read older reports and convert
+
+import (
+	"strings"
+	"time"
+
+	"github.com/ugorji/go/codec"
+)
+
+// For backwards-compatibility with probes that sent a map of latestControls data
+type bcNode struct {
```

Comment:
```suggestion
type nodeWithLatestControls struct {
```

Just for greater clarity and also to be open for the time when "backwards-compatible nodes" might diverge along more than just one property.

Reply:
Actually never mind, I misinterpreted the intention here and forgot that this lives in `backcompat.go` :)